org.gnome.Boxes.Extension.OsinfoDb.json

```diff
@@ -2,7 +2,7 @@
     "id": "org.gnome.Boxes.Extension.OsinfoDb",
     "runtime": "org.gnome.Boxes",
     "runtime-version": "stable",
-    "sdk": "org.gnome.Sdk//43",
+    "sdk": "org.gnome.Sdk//44",
```

Comment:
I think we should bump this directly to 45, so that the Sdk version here matches the runtime version of Boxes https://github.com/flathub/org.gnome.Boxes/blob/master/org.gnome.Boxes.json#L4

this shouldn't make a big difference, but chances are that users that have the 45 runtime installed will likely have the Sdk of the same version installed (if so).

Reply:
Agree. I had to install GNOME 44 sdk for no apparent reason.